.luacheckrc

```diff
@@ -0,0 +1,10 @@
+-- Ignore files copied from GitHub
+exclude_files = {
+    "src/snowplow/lib/escape.lua",
```

Comment:
This file is probably not necessary to exclude, luacheck passes even if I remove it from here.